packages/indicators/src/__tests__/Builder/ArithmeticBuilder/testBuildAnalytics.ts

```diff
@@ -71,35 +72,31 @@ export const testCalculations = () => {
       ],
     ];
 
-    it.each(testData)('%s', (_, formula, expected) => {
-      const config = { formula, aggregation: 'FINAL_EACH_YEAR' };
+    it.each(testData)('%s', (_, formula, expectedValues) => {
+      const config = { formula, aggregation: 'RAW' };
+      const indicator = { code: 'TestAnalytics', builder: 'arithmetic', config };
+      const builder = new ArithmeticBuilder(indicator);
+      const expected = valuesToAnalytics(expectedValues);
 
-      return expect(
-        buildArithmetic({ api, config, fetchOptions: {} }),
-      ).resolves.toIncludeSameMembers(
-        expected.map(value => ({ organisationUnit: 'TO', period: '2019', value })),
-      );
+      expect(
+        builder.buildAnalytics(populatedAnalyticsRepo, {}, {} as FetchOptions),
+      ).toIncludeSameMembers(expected);
     });
   });
 
   describe('multiple orgUnit/period combos', () => {
-    const api = {
-      getAggregator: () => createAggregator(ANALYTIC_RESPONSE_FIXTURES),
-      buildAnalyticsForIndicators: async () => [],
-    };
-
-    const testData: [string, ArithmeticConfig, AnalyticValue[]][] = [
+    const testData: [string, DbRecord, [string, string, number][]][] = [
       [
         'all data elements are defined in all combos',
         {
           formula: 'A_ToPg_20192020 + B_ToPg_20192020',
           aggregation: 'FINAL_EACH_YEAR',
         },
         [
-          { organisationUnit: 'TO', period: '2019', value: 3 },
-          { organisationUnit: 'TO', period: '2020', value: 30 },
-          { organisationUnit: 'PG', period: '2019', value: 3.3000000000000003 },
-          { organisationUnit: 'PG', period: '2020', value: 33 },
+          ['TO', '2019', 3],
```

Comment:
This is a nice example of when the values are self describing enough, that the field names really are just clutter - I like the array format :-)

Reply:
Same here! The object format seems handy for passing analytics around, but array seems simpler for defining them!